propulate/propagators/__init__.py

```diff
@@ -0,0 +1,26 @@
+"""
+This package holds all Propagator subclasses including the Propagator itself.
+"""
+
+__all__ = [
+    "Propagator",
+    "Stochastic",
+    "Conditional",
+    "Compose",
+    "PointMutation",
+    "RandomPointMutation",
+    "IntervalMutationNormal",
+    "MateUniform",
+    "MateMultiple",
+    "MateSigmoid",
+    "SelectMin",
+    "SelectMax",
+    "SelectUniform",
+    "InitUniform",
+    "pso",
+    "StatelessPSO",
+]
+
+from propulate.propagators.propagators import *
+import pso
+from pso.stateless_pso import StatelessPSO
```

Comment:
Because this guy works in contrast to the other with ``Indivisual``s and not with ``Particle``s. 
To emphasise the difference, it is imported directly in the parental package.